src/eat.ts

```diff
@@ -0,0 +1,3 @@
+export default function eat(this: any[]) {
+  return this.reduce(acc => acc, []);
```

Comment:
I love this; the issue here is that it will return a new array and not operate on the array itself so that

```
x = [1,2,3]
y = array.eat()
x === y // false
```

You could use splice here and splice out the whole array and then that would fix that.

```
export default function eat(this: any[]) {
  this.splice(0, this.length)
  return this
}
```

Very serious. Thank you for your seriousness. If you can change that we can add your very serious method.

Reply:
Sorry for not being very serious. I corrected my wrongdoing, I shall now take a vow of silence to show my commitment to the success of loldash.